modules/open_data_schema_dcat/DcatValidator.php

```diff
@@ -1,11 +1,9 @@
 <?php
 
-include __DIR__ . '/../../autoload.php';
-$module_path = drupal_get_path('module', 'open_data_schema_map_xml_output');
-include implode('/', array($module_path, 'autoload.php'));
-
-use JsonSchema\Uri\UriRetriever;
```

Comment:
@kducharm In our current D8 work, I believe the standard is to always use "use statements" instead of having full class names. I assume this is not enforced by codesniffer in D7, but I wonder if we should decide on a standard and add it to codesniffer. Do you have a preference? The only advantage of use statements is that is gives some visibility at the top of what classes are being used, plus it allows you to not repeat yourself if using the same class in multiple places.

Reply:
I'm into having the "use" statements - what was odd in this case was that I could not get it to work having just "use" and not fully qualifying the path when creating an object... it wouldn't hurt to have both though in this case if that's OK w/you